packages/plugin-ext/src/plugin/status-bar-message-registry.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2018 Red Hat, Inc. and others.
+ * Copyright (C) 2018-2022 Red Hat, Inc. and others.
```

Comment:
Minor: I believe we've stopped updating these headers, just keeping the original date in there. Do you mind reverting the changes to the file headers?

Reply:
@msujew Thanks for the quick review!
Sure, I reverted the headers and pushed an update.